Makefile

```diff
@@ -302,3 +302,28 @@ ci.e2e.ansible:
 	make -C test/e2e/ansible e2e.setup
 	WHAT="make -C test/e2e/ansible e2e.run" DOCKER_DEVKIT_DEFAULT_ARGS="--rm --net=host" make devkit.run
 	make -C test/e2e/ansible e2e.clean
+
+
+VERSION = $(shell git describe --tags)
+IMAGE_REPO = mesosphere
+IMAGE_NAME = konvoy-image-builder
+
+release-bundle-GOOS:
+	GOOS=$(GOOS) go build -tags EMBED_DOCKER_IMAGE \
+		-ldflags="-X github.com/mesosphere/konvoy-image-builder/pkg/version.version=$(VERSION)" \
+		-o "./dist/bundle/konvoy-image-bundle-$(VERSION)_$(GOOS)/konvoy-image" ./cmd/konvoy-image-wrapper/main.go
+	cp -a ansible "./dist/bundle/konvoy-image-bundle-$(VERSION)_$(GOOS)/"
+	cp -a goss "./dist/bundle/konvoy-image-bundle-$(VERSION)_$(GOOS)/"
+	cp -a images "./dist/bundle/konvoy-image-bundle-$(VERSION)_$(GOOS)/"
+	cp -a overrides "./dist/bundle/konvoy-image-bundle-$(VERSION)_$(GOOS)/"
+	cp -a packer "./dist/bundle/konvoy-image-bundle-$(VERSION)_$(GOOS)/"
+	cd ./dist/bundle && tar cf konvoy-image-bundle-$(VERSION)_$(GOOS).tar.gz konvoy-image-bundle-$(VERSION)_$(GOOS)
+	rm -rf "./dist/bundle/konvoy-image-bundle-$(VERSION)_$(GOOS)/"
+
+embed-image:
```

Comment:
```suggestion
$(REPO_ROOT_DIR)/cmd/konvoy-image-wrapper/image/konvoy-image-builder.tar.gz:
```

Reply:
Eventually would be good to embed the version in the filename, so that its tracked in make as a dep appropriatly.